proposals/NNNN-high-level-interface-views-and-controllers.md

```diff
@@ -0,0 +1,229 @@
+# High level interface: Views and Controllers
+
+* Proposal: [SDL-NNNN](NNNN-high-level-interface-views-and-controllers.md)
+* Author: [Kujtim Shala](https://github.com/kshala-ford)
+* Status: **Awaiting Review**
+* Impacted Platforms: [ iOS ]
+
+## Introduction
+
+This proposal is based on #448 adding views and view controllers to the high level interface. It proposes a solution to mimic the UI framework of the native OS SDKs.
```

Comment:
Can you please include a link to [SDL 0156](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0156-high-level-interface-foundation.md) ?